diff --git a/test/LanguageServer.Engine.Tests/TestFiles/Test1.xml b/test/LanguageServer.Engine.Tests/TestFiles/Test1.xml
index aa2e71c..3f5d6a9 100644
--- a/test/LanguageServer.Engine.Tests/TestFiles/Test1.xml
+++ b/test/LanguageServer.Engine.Tests/TestFiles/Test1.xml
@@ -4,4 +4,5 @@
         <Element4 />
     </Element2>
     <Element5 ></Element5>
+<Element6 />
 </Element1>
diff --git a/test/LanguageServer.Engine.Tests/XSParserTests.cs b/test/LanguageServer.Engine.Tests/XSParserTests.cs
index bc9c5ac..fe70560 100644
--- a/test/LanguageServer.Engine.Tests/XSParserTests.cs
+++ b/test/LanguageServer.Engine.Tests/XSParserTests.cs
@@ -36,7 +36,7 @@ namespace MSBuildProjectTools.LanguageServer.Tests
         /// <param name="expectedNodeCount">
         ///     The expected number of nodes to be 
         /// </param>
-        [InlineData("Test1", 12)]
+        [InlineData("Test1", 14)]
         [Theory(DisplayName = "XSParser discovers node count ")]
         public void NodeCount(string testFileName, int expectedNodeCount)
         {
@@ -153,7 +153,7 @@ namespace MSBuildProjectTools.LanguageServer.Tests
         /// <param name="endColumn">
         ///     The node's ending column.
         /// </param>
-        [InlineData("Test1", 0, 1, 1, 7, 12)]
+        [InlineData("Test1", 0, 1, 1, 8, 12)]
         [InlineData("Test1", 1, 1, 11, 2, 5)]
         [InlineData("Test1", 2, 2, 5, 5, 16)]
         [InlineData("Test1", 3, 2, 15, 2, 34)]
